plugins/core/base/routes/search.js

```diff
@@ -1,16 +1,33 @@
-const Boom = require('boom');
-const Joi = require('joi');
+const Boom = require("boom");
+const Joi = require("joi");
 
 module.exports = {
-  path: '/search',
-  method: 'POST',
+  path: "/search",
+  method: "GET",
   options: {
     validate: {
-      payload: Joi.object().required()
+      query: {
+        limit: Joi.number().optional(),
+        bookmark: Joi.string().optional(),
+        tool: Joi.alternatives()
+          .try(Joi.array().items(Joi.string()), Joi.string())
+          .optional(),
+        createdBy: Joi.string().optional(),
+        department: Joi.string().optional(),
+        publication: Joi.string().optional(),
+        active: Joi.boolean().optional(),
+        searchString: Joi.string().optional()
+      }
     },
-    tags: ['api', 'editor']
+    tags: ["api", "editor"]
   },
   handler: async (request, h) => {
-    return request.server.methods.db.item.search(request.payload);
+    // Creates new object filterProperties which contains all properties but bookmark and limit
+    const { bookmark, limit, ...filterProperties } = request.query;
+    return request.server.methods.db.item.search(
+      filterProperties,
+      request.query.limit,
```

Comment:
can you use `limit` and `bookmark` here as it's already extracted from `request.query`. Makes it more clear imo.

Reply:
Sure, fixed in https://github.com/nzzdev/Q-server/pull/111/commits/cdcbe7073ae3938d278d9779acca17b0121c2695